core/templates/pages/exploration-editor-page/translation-tab/audio-translation-bar/audio-translation-bar.directive.spec.ts

```diff
@@ -509,9 +509,10 @@ describe('Audio translation bar directive', function() {
     spyOn(siteAnalyticsService, 'registerSaveRecordedAudioEvent');
     spyOn(alertsService, 'addSuccessMessage');
     spyOn(stateRecordedVoiceoversService.displayed, 'addVoiceover');
-    spyOn(assetsBackendApiService, 'saveAudio').and.returnValue($q.resolve({
-      duration_secs: 90
-    }));
+    spyOn(assetsBackendApiService, 'saveAudioAsync')
+      .and.returnValue($q.resolve({
+        duration_secs: 90
+      }));
```

Comment:
Nit(optional):
```suggestion
        spyOn(assetsBackendApiService, 'saveAudioAsync').and.returnValue(
           $q.resolve({duration_secs: 90}));
```

_Follow this elsewhere below._

Reply:
I have tried to test it manually and it works well as far as I know. I will fix the nit comment and hopefully I understood it right.